src/Calendar.js

```diff
@@ -146,7 +147,13 @@ const Calendar = ({
     const allDays = getViewMonthDays(isNewMonth);
     return allDays.map(({ id, value: day, month, year, isStandard }) => {
       const dayItem = { day, month, year };
-      const isDisabled = disabledDays.some(disabledDay => isSameDay(dayItem, disabledDay));
+      let isDisabled = disabledDays.some(disabledDay => isSameDay(dayItem, disabledDay));
```

Comment:
I already have a "isBeforeDate" utility function which is imported, and you can check use that to compare today and days.

Reply:
Thank you Kiarash for pointing that. I'll make the changes tonight.

Reply:
I found some problem with `dayItem` in `renderMonthDays`. for rendering days of week correctly it has some empty days that only have a day:
![Screenshot from 2019-06-11 18-14-54](https://user-images.githubusercontent.com/8453360/59277357-54022600-8c75-11e9-83af-b8ff63ce5e89.png)

so if I use `isBeforeDate` to compare these days, it would render those empty days like this: 
![Screenshot from 2019-06-11 18-19-41](https://user-images.githubusercontent.com/8453360/59277487-8a3fa580-8c75-11e9-818a-67e9ec1db8c4.png)

how should we handle this?

Reply:
There's a `isStandard` prop set to `true` for the visible days. so any `!isStandard` expression can be used to identify the hidden days.

Reply:
I think it's not working as expected. I used the `month` variable and its working actually. If you think we have to use `isStandard` I can start to debug it.

Reply:
I think you're checking `dayItem` variable, but you gotta use the destructured `isStandard` key. It's already there:
![image](https://user-images.githubusercontent.com/20098648/59346646-01357680-8d28-11e9-97df-d1b54509b5a3.png)